src/components/ConnectionHelp.js

```diff
@@ -289,6 +306,25 @@ class ConnectionHelp extends Component {
                 />
               ) : (null)
             }
+            <PaddedBox display="flex" justifyContent="center">
```

Comment:
What would you think about moving this up inside the `rover` conditional (after line 289), so that it only shows up when the "Congrats! Let's go..." button shows up? Make the succeed once before they turn off the guide.

Reply:
I'm afraid there will be users that don't need it at all since they've been instructed what to do and won't know how to turn it off

Reply:
That makes sense.

Could we put it down in a `<DialogActions>` to add a divider that separates it from the rest of the flow? https://material-ui.com/components/dialogs/#confirmation-dialogs